ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/image/v1/image/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,107 @@
+        <items jcr:primaryType="nt:unstructured">
+            <tabs
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/tabs"
+                    maximized="{Boolean}true">
+                <items jcr:primaryType="nt:unstructured">
+                    <basic
+                            jcr:primaryType="nt:unstructured"
+                            jcr:mixinTypes="[granite:InternalArea]"
+                            jcr:title="Basic"
+                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured">
+                            <hideTitle
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:hideResource="true"/>
+                            <placeholder
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:hideResource="true"/>
+                            <bindref
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:hideResource="true"/>
+                            <description
+                                    granite:class="cmp-adaptiveform-image__description"
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="visible"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                    fieldLabel="Description"
+                                    name="./jcr:description"/>
+                            <file
+                                    granite:class="cmp-adaptiveform-image-file-upload"
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="visible"
+                                    sling:resourceType="cq/gui/components/authoring/dialog/fileupload"
+                                    class="cq-droptarget"
+                                    fileNameParameter="./fileName"
+                                    fileReferenceParameter="./fileReference"
+                                    mimeTypes="[image/gif,image/jpeg,image/png,image/tiff,image/svg+xml]"
+                                    name="./file"/>
+                            <alt
+                                    granite:class="cmp-adaptiveform-image-alt"
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="visible"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                    fieldLabel="Alternate Text"
+                                    name="./altText"/>
+                            <excludeFromDor
```

Comment:
I dont think this property is required

Reply:
it was there in old form , i replicated same , do we need to remove?